### PR TITLE
(#826) The All Computers (Automatic Group) Group

### PR DESCRIPTION
**Pull request by st3phhays**
## Description Of Changes
This adds information around the new All Computers (Automatic Group) Group that is now created by default in CCM 0.11.0.

## Motivation and Context
This was added as part of Chocolatey Central Management 0.11.0. 

## Testing

* [x] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made

* [ ] Minor documentation fix (typos etc.).
* [x] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [ ] New documentation page added.
* [ ] The change I have made should have a video added, and I have raised an issue for this.
    * Issue #

## Change Checklist

* [ ] Requires a change to menu structure (top or left hand side)/
* [ ] Menu structure has been updated

## Related Issue
#826

### CODE REVIEW

File: input/en-us/central-management/usage/website/groups.md [hunk 1]
```diff
@@ -17,6 +17,20 @@ If you do not see this menu entry, verify with your administrator whether your C
 
 ![Groups menu entry on the CCM Dashboard](/assets/images/groups/ccm-groups-menu.png)
 
+## The All Computers (Automatic Group)
+
+> :choco-info: **NOTE**
+>
+> This feature became available starting with Chocolatey Central Management 0.11.0.
```
Comment (corbob) [inline input/en-us/central-management/usage/website/groups.md:24]: Should this note be a partial that we can use for all of the places we've added similar notes? I know I have a slightly differently worded one in #832 ...

File: input/en-us/central-management/usage/website/groups.md [hunk 2]
```diff
@@ -17,6 +17,20 @@ If you do not see this menu entry, verify with your administrator whether your C
...
+
+All Computers are automatically added to a system managed group named `All Computers (Automatic Group)`. This Group can not be edited or deleted.
+
+![The All Computers (Automatic Group) on the Groups page with an arrow pointing to the Group in the table](/assets/images/groups/ccm-groups-automatic-group-table-entry.png)
+
+The  `All Computers (Automatic Group)` will have a blue lock icon next to the it's name in the table. Hovering over this lock icon will display a tooltip with more information.
```
Comment (corbob) [inline input/en-us/central-management/usage/website/groups.md:30]:
```suggestion
The  `All Computers (Automatic Group)` will have a blue lock icon next to it's name in the table. Hovering over this lock icon will display a tooltip with more information.
```

**Review by corbob — CHANGES_REQUESTED** [review]

File: input/en-us/central-management/usage/website/groups.md (hunk 1 above)
Comment (st3phhays) [reply input/en-us/central-management/usage/website/groups.md]: I have changed mine to what is is in #832, and fixed up the other ones that are not consistent in another commit.

File: input/en-us/central-management/usage/website/groups.md [hunk 3]
```diff
@@ -17,6 +17,20 @@ If you do not see this menu entry, verify with your administrator whether your C
...
+>
+> This feature is available in Chocolatey Central Management starting with version 0.11.0.
+
+All Computers are automatically added to a system managed group named `All Computers (Automatic Group)`. This Group can not be edited or deleted.
+
+![The All Computers (Automatic Group) on the Groups page with an arrow pointing to the Group in the table](/assets/images/groups/ccm-groups-automatic-group-table-entry.png)
+
+The  `All Computers (Automatic Group)` will have a blue lock icon next to the the name in the table. Hovering over this lock icon will display a tooltip with more information.
+
+![The All Computers (Automatic Group) on the Groups page with an arrow pointing to the tooltip containing additional information](/assets/images/groups/ccm-groups-automatic-group-tooltip.png)
```
Comment (corbob) [inline input/en-us/central-management/usage/website/groups.md:32]: 🤔 Should the alt-text here include the text that is in the tooltip as additional information?

**Review by corbob — APPROVED** [review]
LGTM!

File: input/en-us/central-management/usage/website/groups.md [hunk 4]
```diff
@@ -17,6 +17,20 @@ If you do not see this menu entry, verify with your administrator whether your C
 
 ![Groups menu entry on the CCM Dashboard](/assets/images/groups/ccm-groups-menu.png)
 
+## The All Computers (Automatic Group)
+
+> :choco-info: **NOTE**
+>
+> This feature is available in Chocolatey Central Management starting with version 0.11.0.
+
+All Computers are automatically added to a system managed group named `All Computers (Automatic Group)`. This Group can not be edited or deleted.
```
Comment (vexx32) [inline input/en-us/central-management/usage/website/groups.md:26]:
```suggestion
All Computers present in Chocolatey Central Management are automatically added to a system managed group named `All Computers (Automatic Group)`. This Group can not be edited or deleted.
```

I think it's worth being just a tiny bit more explicit here, I kinda missed this was mentioned at all on first scan.

**Review by vexx32 — APPROVED** [review]
One minor thing I think we could maybes make a tad more clear, but otherwise this looks great. Nice work @st3phhays! :3

File: input/en-us/central-management/usage/website/groups.md (hunk 4 above)
Comment (st3phhays) [reply input/en-us/central-management/usage/website/groups.md]: I have fixed this up.

**Review by vexx32 — APPROVED** [review]

**Pull request merged**